### PR TITLE
Support http-wire-trace for debugging

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

File: lib/fluent/plugin/in_s3.rb [hunk 1]
```diff
@@ -185,19 +190,30 @@ def create_s3_client
       options = setup_credentials
       options[:region] = @s3_region if @s3_region
       options[:proxy_uri] = @proxy_uri if @proxy_uri
+      log.trace do
```
Comment (repeatedly) [inline lib/fluent/plugin/in_s3.rb]: How about `if log.level == Fluent::Log::LEVEL_TRACE`?

Comment (tagomoris) [inline lib/fluent/plugin/in_s3.rb]: `Fluent::Log#on_trace` method is defined for such case.

Comment (repeatedly) [inline lib/fluent/plugin/in_s3.rb]: Ah, I forgot it.

Comment (okkez) [inline lib/fluent/plugin/in_s3.rb]: Ah, I forgot `Fluent::Log#on_trace`. Thanks!

**Pull request merged**